euth/actions/models.py

```diff
@@ -43,6 +43,8 @@ class Action(models.Model):
     verb = models.CharField(max_length=255, db_index=True)
     description = models.TextField(blank=True, null=True)
 
+    recipients = models.TextField(blank=True, null=True)
```

Comment:
I wanted to remove the logic of getting the recipients from the actions app. The recipients are not always the same, e.g. for 'create': for comments it is the creator of the resource that has been commented but for idea it is the moderators of the project.